appveyor.yml

```diff
@@ -28,6 +28,7 @@ build: false
 
 test_script:
   - pip install -e .[tests]
+  - pip install pandas==0.23.4
```

Comment:
please add a comment here as to why this is needed

Reply:
done